### ISSUE TITLE
How to make the robotic arm reach a fixed position at a specified time

### ISSUE THREAD

**Issue by zzzbbs** [ros-planning/moveit#3289]
Hi, 
I have a question while using moveit planning interface. I want the robotic arm to arrive at a certain position at a fixed time, but the plan provided by moving it seems to have no interface for specifying the arrival time. Could you give me some suggestions about this.Thanks~

**Comment by zzzbbs** [comment]
I found a easy solution, which may not work well on real time motion planning using moveit. Firstly, I set the target position of what I want. Secondly, I used plan() function to get a RobotTrajectory. Because I don't need to work on real time, I used the position of other joints only. I extract the position from trajectory file, and interpolate the position series to the time of what I want. Finally, I transform the interpolated trajectory to the RobotTrajectory format, used moveit to actuate the motor to the target position in duration. The most disadvantage of this method is that it cannot work on the real time environment.
